CorsixTH/Lua/world.lua

```diff
@@ -1370,6 +1390,7 @@ function World:nextEarthquake()
     self.next_earthquake.size = control.Severity
     self.next_earthquake.remaining_damage = self.next_earthquake.size
     self.next_earthquake.damage_timer = earthquake_damage_time
+    self.next_earthquake.warning_timer = earthquake_warning_period
```

Comment:
I set it to 0 in afterLoad because any save game with an active earthquake the actual earthquake is already in progress (warning period is over).